Website/src/items/templates/edgewebsite/cmp entities/Edge Speaker/Social/Twitter.yml

```diff
@@ -0,0 +1,38 @@
+﻿---
+ID: "2a659163-cd5a-4026-8256-d2acc7ab11c6"
+Parent: "b82dfeac-780e-45eb-a121-d8252c4ec4e2"
+Template: "455a3e98-a627-4b40-8035-e683a0331ac7"
+Path: /sitecore/templates/Project/edgewebsite/cmp entities/Edge Speaker/Social/Twitter
```

Comment:
This field should be appended with "...ProfileLink" with a display name of "... Profile Link".